swap/src/main.rs

```diff
@@ -12,12 +12,18 @@
 #![forbid(unsafe_code)]
 #![allow(non_snake_case)]
 
-use crate::cli::{Command, Options, Resume};
+use crate::{
+    cli::{Command, Options, Resume},
+    config::{
+        initial_setup, query_user_for_initial_testnet_config, read_config, ConfigNotInitialized,
+    },
+};
 use anyhow::{Context, Result};
```

Comment:
```suggestion
use cli::{Command, Options, Resume};
use config::{
        initial_setup, query_user_for_initial_testnet_config, read_config, ConfigNotInitialized,
    };
use anyhow::{Context, Result};
```
The usage of `use crate::` in the `main.rs` is strange (I know it was here before).

Reply:
This is because of the lib as well. `config` is ambiguous, because there is we have a dependency named `config` as well, and `swap::config` causes compile issues.